src/components/DatePicker/DatePicker.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <v-form :attach="attach" ref="form" class="date-picker-form">
+    <v-menu v-model="displayPicker"
+            :close-on-click="false"
+            :close-on-content-click="false"
+            :nudge-top="nudgeTop"
+            :nudge-bottom="nudgeBottom"
+            :nudge-left="nudgeLeft"
+            :nudge-right="nudgeRight"
+            transition="scale-transition"
+            offset-y
+            bottom
+            min-width="290"
+    >
+      <template v-slot:activator="{ on }">
+        <v-text-field id="date-text-field"
```

Comment:
Will you make the text field editable?

Reply:
I asked Tracey this when making it, and she said no at the time. 
Is this a different requirement for FED?

Reply:
I don't know either way, so "no" until it's "yes".